daemon/loadbalancer.go

```diff
@@ -335,8 +335,7 @@ func openServiceMaps() error {
 
 // SyncLBMap syncs the bpf lbmap with the daemon's lb map. All bpf entries will overwrite
 // the daemon's LB map. If the bpf lbmap entry has a different service ID than the
-// KVStore's ID, that entry will be updated on the bpf map accordingly with the new ID
-// retrieved from the KVStore.
+// KVStore's ID, that entry will be removed.
```

Comment:
Since kvstore is no longer used, update this comment.

Reply:
Will address in a separate PR.